src/frontend/src/components/forms/fields/ApiFormField.tsx

```diff
@@ -216,6 +217,8 @@ export function ApiFormField({
         );
       case 'date':
         return <DateField controller={controller} definition={definition} />;
+      case 'datetime':
+        return <DateField controller={controller} definition={definition} />;
```

Comment:
This could be simplified to using two case blocks with one return as both have the same arguments like done with int, float, ...